activerecord/lib/active_record/railtie.rb

```diff
@@ -26,6 +26,9 @@ class Railtie < Rails::Railtie # :nodoc:
     config.active_record.use_schema_cache_dump = true
     config.active_record.maintain_test_schema = true
 
+    config.active_record_sqlite3adapter = ActiveSupport::OrderedOptions.new
+    config.active_record_sqlite3adapter.represent_boolean_as_integer = nil
```

Comment:
Huh, I thought it used `respond_to?`. Don't our other `set_configs` initializers do that?